dev-libs/libphonenumber/libphonenumber-8.12.26.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
`# Copyright 2021 Gentoo Authors`

Reply:
I have always used 1999-2021 as both skel.ebuild and header.txt reports it

Reply:
Use real initial year when it is known.